docs/source/examples/zne_logical_rb_cirq_stim.md

```diff
@@ -0,0 +1,303 @@
+This tutorial also introduces the use of Mitiq's ZNE functions with Cirq as the frontend and the [Stim](https://github.com/quantumlib/Stim) stabilizer simulator as the backend.
+
++++
+
+## Setup
+
++++
+
+Since Stim is not a Mitiq-supported frontend, and the [stimcirq](https://pypi.org/project/stimcirq/) library has a conversion utility for Cirq and Stim circuits, we will use Cirq as the frontend for this example.
+
+```{code-cell} ipython3
+import cirq
+import matplotlib as mpl
+import numpy as np
+import stim
+import stimcirq
+import mitiq
+```
+
+In this example, we assume a physical error rate of 0.004 and a threshold of 0.009.
+Since the physical error rate is below the threshold, the computation is operating in the fault-tolerant regime.
+
+```{code-cell} ipython3
+n_qubits = 2
+p_err = 0.004
+p_th = 0.009
+base_shots = 10**4
+device_size = 1500
+```
+
+Randomized benchmarking is typically performed at the physical level, but it can be applied at the logical level as shown in Ref {cite}`Combes_2017_arXiv_logical_rb`.
+The modularity of logical RB circuits allows for parallelization of circuit executions, as permitted by the size of the device and number of physical qubits required for the logical circuit.
+To generate the RB circuits, we use a built-in Mitiq function, 
+{func}`.benchmarks.randomized_benchmarking.generate_rb_circuits()`, where the `n_cliffords` argument refers to the number of Clifford groups in the circuit, and therefore scales the depth of the circuit. 
+In this example the Clifford depth is 100, which would be around the limit of classical simulability if the operations in the circuit were instead _non-Clifford_.
+
+```{code-cell} ipython3
+:tags: [skip-execution]
+trials = 10
+n_cliffords = 100
+cirq_circuits = mitiq.benchmarks.generate_rb_circuits(
+    n_qubits, n_cliffords, trials
+)
+```
+
+```{code-cell} ipython3
+:tags: [remove-cell]
+
+# hidden cell for faster docs build
+trials = 5
+n_cliffords = 10
+p_err = 0.006  # generate meaningful plot at lower depth for diagnostics
+cirq_circuits = mitiq.benchmarks.generate_rb_circuits(
+    n_qubits, n_cliffords, trials
+)
+```
+
+We also fill the idle windows of the circuit with identity gates, to which we will later append append $X$ and $Z$ errors, as each moment of the circuit corresponds to one correction cycle.
+
+```{code-cell} ipython3
+def fill_circuit_with_identities(circ):
+    qubits = circ.all_qubits()
+    return cirq.Circuit([
+        m.expand_to(qubits) for m in circ
+    ])
+```
+
+```{code-cell} ipython3
+filled_circuits = []
+for c in cirq_circuits:
+    filled_circuit = fill_circuit_with_identities(c)
+    filled_circuit.append(cirq.measure(filled_circuit.all_qubits()))
+    filled_circuits.append(filled_circuit)
+```
+
+## Noise model
+
++++
+
+The noise is modeled as single-qubit $X$ and $Z$ errors, with probability $p_L$ given by an empirical formula from Ref. {cite}`Fowler_2012_PRA`: 
+
+$$
+p_L\cong 0.03(p/p_{th})^{(d+1)/2}
```

Comment:
Since this is already in display mode, might as well do a full standing fraction. Also making $\mathrm{th}$ romanized for consistency with original source.
```suggestion
p_L\cong 0.03(\frac{p}{p_\mathrm{th}})^{(d+1)/2}
```